includes/class-wc-cart.php

```diff
@@ -430,7 +430,7 @@ public function set_totals( $value = array() ) {
 	 * @param string $value Value to set.
 	 */
 	public function set_subtotal( $value ) {
-		$this->totals['subtotal'] = wc_format_decimal( $value, wc_get_price_decimals() );
+		$this->totals['subtotal'] = $value;
```

Comment:
Nice catch. We did a similar change sometime back, and while no issues have been reported so far because of it, there could have been.

For now, I have restored the formatting changes, but have removed rounding, so it's backward compatible.